imagej/imagej.py

```diff
@@ -321,6 +472,37 @@ def from_java(self, data):
                 raise exc
             return to_python(data)
 
+        def _dataset_to_xarray(self, dataset):
+            """
+            Converts an ImageJ dataset into an xarray
+            :param dataset: ImageJ dataset
+            :return: xarray with reversed (C-style) dims and coords as labeled by the dataset
+            """
+            attrs = self._ij.py.from_java(dataset.getProperties())
+            axes = [(cast('net.imagej.axis.DefaultLinearAxis', dataset.axis(idx)))
+                    for idx in range(dataset.numDimensions())]
+
+            dims = [self._ijdim_to_pydim(axes[idx].type().getLabel()) for idx in range(len(axes))]
+            values = self.rai_to_numpy(dataset)
+            coords = self._get_axes_coords(axes, dims, numpy.shape(numpy.transpose(values)))
+
+            xarr = xr.DataArray(values, dims=list(reversed(dims)), coords=coords, attrs=attrs)
```

Comment:
The `reversed` will need to become a helper method if we add the handling of RGB, right?